modules/integration/test-results-service/src/main/java/org/wso2/sp/tests/eventscollector/VerifyTest.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.sp.tests.eventscollector;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import io.swagger.annotations.Contact;
+import io.swagger.annotations.Info;
+import io.swagger.annotations.License;
+import io.swagger.annotations.SwaggerDefinition;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.wso2.msf4j.Request;
+import org.wso2.sp.tests.eventscollector.exception.TestNotFoundException;
+import java.util.HashMap;
+import java.util.Map;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.HeaderParam;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+
+/**
+ * VerifyTest class.
+ */
+
+@Api(value = "testresults")
+@SwaggerDefinition(
+        info = @Info(
+                title = "SP Test Events Swagger Definition", version = "1.0",
+                description = "Stream Processor Test Events in-memory persisting service",
+                license = @License(name = "Apache 2.0", url = "http://www.apache.org/licenses/LICENSE-2.0"),
+                contact = @Contact(
+                        name = "WSO2 Pvt Ltd",
+                        email = "analytics-team@wso2.com",
+                        url = "http://wso2.com"
+                ))
+)
+
+@Path("/testresults")
+public class VerifyTest {
+
+    private static final Logger log = LoggerFactory.getLogger(VerifyTest.class);
+    private Map<String, Event> testResults = new HashMap<>();
+
+    public VerifyTest() {
+
+    }
```

Comment:
If there's nothing in the constructor, you can remove it. By default, there's an empty constructor.